wiki/en/Getting-Started.md

```diff
@@ -69,7 +69,7 @@ Now use the “Connect” button in the Jamulus main window to join a Server. A
 
 The most important thing about Servers is their “ping time”. The bigger the number, the harder it will be to play in time with others. Usually, you would select a Server with a ping of less than 50ms if you can.
 
-Once connected to a Server, make sure you can hear yourself OK and fix any input volume or other problems. You can have a look at the [troubleshooting page](/wiki/Client-Troubleshooting) for common problems. And of course, check that you are also able to obey [The Golden Rule](/wiki/Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together).
+Once connected to a Server, make sure you can hear yourself OK and fix any input volume or other problems. You can have a look at the [troubleshooting page](/wiki/Client-Troubleshooting) for common problems. Also check that you are also able to obey [The Golden Rule](/wiki/Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together).
 
 ## Playing for the first time
 
```

Comment:
There's only one change I'd agree with in all of these.

Reply:
If you mean https://github.com/jamulussoftware/jamuluswebsite/pull/827/files/b0789fbad59a2f0e184ea6574054ba724f89d2b5#r977239188...

1) Why would a PR that _only_ added **or** not be worth doing?
2) Is there a scenario where **or** could be added?